persistence-utils-demo/pom.xml

```diff
@@ -23,7 +23,7 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
```

Comment:
if you check the output of persistence-utils-demo.iar -> lib/mvn-deps you'll see that a bunch of thirdparty-libraries in included in the IAR as well. I think these are not required? or not all of it?

please check them:
- if we do not require hibernate-jpamodelgen at runtime, please switch its' scope to provided (so these libs will not be inclueded in the final ivy-archive)
- otherwise, try to exclude some dependencies that are already on the classpath: I'd guess that <exclusions><groupId>*</groupId><artifactId>*</artifactId>... will already do what we want